scene/2d/animated_sprite.cpp

```diff
@@ -355,23 +355,32 @@ void AnimatedSprite::_notification(int p_what) {
 		case NOTIFICATION_DRAW: {
 
 			if (frames.is_null()) {
-				print_line("no draw no faemos");
+				print_line("no draw no faemos: " + String(get_path()));
 				return;
 			}
 
 			if (frame < 0) {
-				print_line("no draw frame <0");
+				print_line("no draw frame <0: " + String(get_path()));
```

Comment:
This also is not explicit.

Reply:
Its not explicit, but it is the phrasing that was already being used.

Reply:
Sure, not implying it's your fault, just pointing it out so other reviewers can see it.

Reply:
I understand, just wanted to give my justification for not changing it.  I don't know if someone else really needs that phrasing.